ocs_ci/templates/baremetal-pxefile/01-0c-c4-7a-f3-ea-5e

```diff
@@ -0,0 +1,11 @@
+DEFAULT menu.c32
```

Comment:
@prsurve 
Most of the pxe files found here looks like duplicates , only thing changes is the filename which is MAC address (correct me if I am missing something). Why can't we have a list of bm MAC addresses stored in an internal file (may be in google drive) and have a single template file in ocs-ci repo. Then we can generate as many pxe files as we want based on MAC list from the internal file, this is also scalable because if you just add new MAC in the private file ocs-ci code should work without any change. Correct me if I am wrong.

Reply:
yes you are correct will add this in next commit

Reply:
Done